drivers/bluetooth/hci/Kconfig.siliconlabs

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2024 SiliconLabs <www.silabs.com/>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_SILABS_EFR32_BUFFER_MEMORY
+	int "Silicon Labs Bluetooth Library memory buffer size"
+	depends on BT_SILABS_EFR32
```

Comment:
I suggest to place all these options under `menuconfig BT_SILABS_EFR32` rather than repeating `depends on BT_SILABS_EFR32`